src/index.js

```diff
@@ -1,5 +1,7 @@
 require('fullcalendar/dist/fullcalendar.js');
 require('fullcalendar/dist/locale-all.js');
+
```

Comment:
```suggestion
```

Reply:
In other modules, we also have this blank line between fullcalendar and below require. So I think we should keep this convention :D